flask_app/backend_jobs/pipeline_utils/database_schema.py

```diff
@@ -52,6 +52,10 @@
 COLLECTION_IMAGES_FIELD_HASHMAP = 'hashmap'
 # Type: member, A VisibilityType member which represents whether the image was verified.
 COLLECTION_IMAGES_FIELD_VISIBILITY = 'visibility'
+# Type: string, the url of the image with a low resolution.
+COLLECTION_IMAGES_FIELD_ADDITIONAL_URL = 'additionalUrl'
+# Type: boolean, whether the given image satisfies minimum requirements.
+COLLECTION_IMAGES_FIELD_PASS_FILTER = 'passFilter'
```

Comment:
this naming is not great. maybe change to `passedFilter` and `COLLECTION_IMAGES_FIELD_PASSED_FILTER`

Reply:
changed